src/codegen/function_body_chunk.rs

```diff
@@ -159,7 +159,7 @@ impl Builder {
                     }
                 }
             }).collect();
-        let result = Chunk::Tuple(result, TupleMode::Simple);
+        let result = Chunk::Tuple(result, TupleMode::WithUnit);
```

Comment:
@antoyo Maybe you have some other reason to uncomment and use `TupleMode::Simple`

Reply:
That should be okay.